src/app/kbv/steps.js

```diff
@@ -34,6 +35,31 @@ module.exports = {
     controller: singleAmountQuestionController,
     next: "load-question",
   },
+  "/prove-identity-another-way": {
+    backLink: null,
+    entryPoint: true,
+    fields: ["abandonRadio"],
+    controller: proveIdentityAnotherWayController,
+    next: [
+      {
+        field: "abandonRadio",
+        value: "stop",
+        next: "/oauth2/callback",
+      },
+      {
+        field: "abandonRadio",
+        value: "continue",
+        next: [
+          {
+            fn: proveIdentityAnotherWayController.prototype
+              .isQuestionJourneyStarted,
```

Comment:
Would it be simpler and less brittle if we just use the existing `loadQuestionController.prototype.hasQuestion` method here instead - if there's a question in the session then they've been to a question page, if there's no question they haven't got past the start page yet. Thoughts @shivanshuit914 @merlinc?

Reply:
The only edge case is if they've been through the start page to a question and then gone browser back to the start page (but we don't have back links so we're not really supporting 'back')